google/cloud/security/common/gcp_api/bigquery.py

```diff
@@ -47,7 +47,7 @@ def get_rate_limiter(self):
         return RateLimiter(FLAGS.max_bigquery_api_calls_per_100_seconds,
                            self.DEFAULT_QUOTA_TIMESPAN_PER_SECONDS)
 
-    def get_bigquery_projectids(self):
+    def get_bigquery_projectids(self, key='projects'):
         """Request and page through bigquery projectids.
 
```

Comment:
Actually, this should not even been arg since it's hardcoded.  Wanted to move it earlier inside the method. but this will make me do it.  :)